README.rst

```diff
@@ -35,7 +35,11 @@ files). It provides an efficiently searchable index of the bulk data files, and
 Installation
 ============
 
-Install ``modelmeta`` using the standard methods for any other Python package.
+Installation is fully automated through ``make``::
+
+    $ make
+
+If you wish to install ``modelmeta`` using the standard methods follow the steps below.
```

Comment:
How about: If you wish to install ``modelmeta`` manually, follow the steps below.